src/main/java/ValidationMessages_fr.properties

```diff
@@ -0,0 +1,49 @@
+user.firstName=Veuillez entrer votre pr�nom.
+user.lastName=Veuillez entrer votre nom de famille. 
+user.invalidEmail=Veuillez entrer une adresse courriel valide.
+user.enterUsername=Veuillez entrer un nom d'utilisateur.
+user.usernameLength=Le nom d'utilisateur doit comporter entre 2 et 60 caract�res.
+user.illegalCharacters=Caract�re(s) non valide(s) utilis�(s). Les caract�res valides sont a-Z, 0-9, '_', '-' et '.'.
```

Comment:
Noticed a grammatical correction on this bundle msg, removing the rogue trailing double quote. This and any other improvements in the French translation should be applied to the English version as well.

Reply:
Hum... I accidentally introduced an error while editing the file. *dataverse.nameIllegalCharacters* variable should be:

dataverse.nameIllegalCharacters=Caractère(s) non valide(s) utilisé(s). Les caractères valides sont a-Z, 0-9, '_', '-'.            *(that is without the period ('.') character)*

instead of 

dataverse.nameIllegalCharacters=Caractère(s) non valide(s) utilisé(s). Les caractères valides sont a-Z, 0-9, '_', '-' et '.'.

(equivalent to the English : dataverse.nameIllegalCharacters=Found an illegal character(s). Valid characters are a-Z, 0-9, '_', and '-'.)

Adding a period ('.') character to a Dataverse alias does throw an error.

@JayanthyChengan can you make the correction?

Reply:
@mhvezina : corrected. please verify.